themes/godotengine/pages/news.htm

```diff
@@ -7,7 +7,7 @@
 [blogPosts]
 pageNumber = "{{ :page }}"
 categoryFilter = "{{ :slug }}"
-postsPerPage = 24
+postsPerPage = 3
```

Comment:
Was this number reduced for testing purposes? Does it affect how the list of news articles displays on <https://godotengine.org/news>?

Reply:
Yes, That was added for debugging purposes. Should be changed back to 24